code/object/objectdock.cpp

```diff
@@ -437,7 +437,7 @@ void dock_move_docked_objects(object *objp)
 			fastest_objp = objp;
 	}
 	
-	// start a tree with that object as the parent... do NOT use the �berfunction for this,
+	// start a tree with that object as the parent... do NOT use the überfunction for this,
```

Comment:
This line shows as modified, but I don't see what changed.

Reply:
I'm not entirely sure what happened there but on my initial commit I saw a change from an "ü" to an invalid character. When I tried to revert my changes nothing happened so I retyped the character in a text editor which fixed the displayed character but still showed some changes.

My guess it that the character was using an encoding different from UTF-8 which explains why it appeared as an invalid character.

Reply:
I thought it might have something to do with the ü but I assumed that if the line was fixed to be identical, the line wouldn't show up as changed in a diff.

Ah well, not a blocker.  LGTM.

Reply:
On second thought, I betcha the original ü was extended ASCII whereas the new ü is proper Unicode.  That would do it.